main/options/options.h

```diff
@@ -273,6 +273,8 @@ struct Options {
 
     std::shared_ptr<FileSystem> fs = std::make_shared<OSFileSystem>();
 
+    std::string parser = "sorbet";
```

Comment:
Let's use an enum instead, you can take a look at how it works for phases: https://github.com/sorbet/sorbet/blob/master/main/options/options.h#L96-L107.